rtos/Kernel.h

```diff
@@ -47,14 +47,14 @@ uint64_t get_ms_count();
 /** Attach a function to be called by the RTOS idle task
   @param   fptr  pointer to the function to be called
 
-  @note You may call this function from ISR context.
+  @note You can call this function from ISR context.
 */
 void attach_idle_hook(void (*fptr)(void));
 
-/** Attach a function to be called when a task is killed
+/** Attach a function to be called when a task is terminated
   @param   fptr  pointer to the function to be called
 
-  @note You may call this function from ISR context.
+  @note You can call this function from ISR context.
```

Comment:
@kjbracey-arm - I think we need to be stricter, so Im changing to "can", please review.

Reply:
The "may" wording is used in lots of other files, so if you want to change it should be consistent. Although it's "cannot" in the same places. I'd say the docs team probably have a global style rule here, so check with them.

I think "may" is clearer for the positive case, personally - it says you're "permitted" to, rather than just being "able" to.

Reply:
The question is for a new user wanting to use this function doesn't get much info from reading this. When we say "may" it sounds like user should go find if its even possible for the specific context he wants to use. I also wonder why someone would want to use these functions from ISR context.

Reply:
@kjbracey-arm ^^^
(It wasn't to clear to me that he had already replied)

Reply:
@kjbracey-arm  - Can you please look at my comment above? You did raise a good point about consistency but I do think that can be addressed in other parts while we can document this better still.